source/material_model/rheology/elasticity.cc

```diff
@@ -325,18 +334,16 @@ namespace aspect
         //
         // We also use this parameter when we are still *before* the first time step,
         // i.e., if the time step number is numbers::invalid_unsigned_int.
-        const double dte = ( ( this->get_timestep_number() > 0 &&
+        double dte = ( ( this->get_timestep_number() > 0 &&
                                this->simulator_is_past_initialization() &&
                                use_fixed_elastic_time_step == false )
                              ?
-                             this->get_timestep()
+                             this->get_timestep() * stabilization_scale_factor
                              :
                              fixed_elastic_time_step);
         return dte;
       }
 
-
-
```

Comment:
please restore these empty lines. our convention is to have three empty lines between function in .cc files, and one empty line between functions in .h files.